package.json

```diff
@@ -55,7 +55,7 @@
         "babel-plugin-react-intl": "^7.5.1",
         "babel-plugin-transform-require-ignore": "^0.1.1",
         "box-node-sdk": "^1.33.0",
-        "box-ui-elements": "^15.0.0-beta.6",
+        "box-ui-elements": "18.1.0",
```

Comment:
Is the `^` removed on purpose? Do we need this exact version of BUIE?

Reply:
Hm good point, I referenced how box-content-preview used BUIE, and it seems like it didn't use `^`, so I followed the same principle. I don't see an issue with keeping it compatible with the version, so we'll do that.

Reply:
The carat auto updates for minor release versions. Not sure if changing this will cause problems down the road as it's had the carat. I'd leave it there.